### DISCUSSION TITLE
Possibility to change Target Branch for Onboarding PR

### DISCUSSION THREAD

**Discussion by Dimmsen** [renovatebot/renovate#20329] [Request Help] [answered]
### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us what version of Renovate you run.

newest?

### If you're self-hosting Renovate, select which platform you are using.

Bitbucket Server

### Was this something which used to work for you, and then stopped?

I never saw this working

### Wanted end result.

In our environment we use one main branch and another develop branch. By default the dependabot creates the onboarding PR with the main branch as target branch. I wanted to ask if it is possible to configure the onboarding PR so that it targets and merges the renovate/configure and the renovate.json into the develop branch? So that I can later on manually merge the develop into the master branch. 

### What you tried so far.

/

### Relevant debug logs

<details><summary>Logs</summary>

```
Copy/paste the relevant log(s) here, between the starting and ending backticks
```

</details>

**Comment by rarkins** [comment, marked as answer]
No, Renovate always needs a config in the default branch. Search this repo for prior conversations mentioning `useBaseBranchConfig`
